src/main.rs

```diff
@@ -43,12 +43,19 @@ struct CLICommand {
 
 #[derive(Subcommand, Debug, Clone)]
 enum Command {
-    VerifyStructure,
+    VerifyStructure(VerifyStructureArgs),
```

Comment:
I would have simply:


```suggestion
    VerifyStructure(usize),
```

Reply:
Or a `u32`

Reply:
Something similar is possible (acc. to clap's derive tutorial).